test/e2e/azure_suite_test.go

```diff
@@ -94,12 +93,21 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(mgmt).NotTo(BeNil())
 
-	// TODO Figure out how to keep these versions in sync across the code base
-	capi := &generators.ClusterAPI{Version: "v0.2.9"}
+	certmanagerYaml := "https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml"
+	mgmt.ApplyYAML(ctx, certmanagerYaml)
+
+	// Wait for CertManager to be available before continuing
+	c, err := mgmt.GetClient()
+	Expect(err).NotTo(HaveOccurred())
+	waitDeployment(c, "cert-manager", "cert-manager-webhook")
+
+	// Deploy the CAPI, CABPK, and KCP components from Cluster API repository,
```

Comment:
what's KCP?

Reply:
its the control plane new feature in cluster-api, but not being used here yet, so I'll fix the comment